toplevel/topdirs.ml

```diff
@@ -447,6 +446,13 @@ let _ = add_directive "ppxs_clear"
              (external and in-process)";
     }
 
+let _ = add_directive "remove_in_process_ppxs"
+    (Directive_none Pparse.clear_in_process_ppx)
+    {
+      section = section_options;
+      doc = "Forget all added in process PPX preprocessors";
+    }
```

Comment:
No no, this is just as bad. What happens once ocamlfind starts to add ppx in-process? I meant that `#remove_in_process_ppx` would take an argument and just that value would be removed.